crates/re_ui/src/modal.rs

```diff
@@ -109,10 +130,16 @@ impl Modal {
 
         let mut open = ui.input(|i| !i.key_pressed(egui::Key::Escape));
 
+        let screen_height = ui.ctx().screen_rect().height();
+        let modal_vertical_margins = (75.0).at_most(screen_height * 0.1);
+
         let mut window = egui::Window::new(&self.title)
```

Comment:
```suggestion
        let mut window = egui::Window::new(&self.title)
            .id(egui::Id::new("modal", &self.title)) // Workaround for https://github.com/emilk/egui/pull/3721 until we make a new egui release
```

Reply:
Works (well—to the extent that it invalidates any existing cache).

Reply:
Yes, that's correct. So we should be good.